corehq/pillows/reportxform.py

```diff
@@ -32,10 +38,31 @@ def transform_xform_for_report_forms_index(doc_dict):
         if domain not in getattr(settings, 'ES_XFORM_FULL_INDEX_DOMAINS', []):
             # full indexing is only enabled for select domains on an opt-in basis
             return None
-        convert_property_dict(doc_ret['form'], REPORT_XFORM_MAPPING['properties']['form'], override_root_keys=['case'])
+        convert_property_dict(doc_ret['form'], REPORT_XFORM_INDEX_INFO.mapping['properties']['form'], override_root_keys=['case'])
         if 'computed_' in doc_ret:
             convert_property_dict(doc_ret['computed_'], {})
 
         return doc_ret
     else:
         return None
+
+
+def get_report_xform_to_elasticsearch_pillow(pillow_id='ReportXFormToElasticsearchPillow'):
+    checkpoint = PillowCheckpoint(
+        'report-xforms-to-elasticsearch',
+    )
+    form_processor = ElasticProcessor(
+        elasticsearch=get_es_new(),
+        index_info=REPORT_XFORM_INDEX_INFO,
+        doc_prep_fn=transform_xform_for_report_forms_index
+    )
+    kafka_change_feed = KafkaChangeFeed(topics=[topics.FORM, topics.FORM_SQL], group_id='report-forms-to-es')
+    return ConstructedPillow(
+        name=pillow_id,
+        checkpoint=checkpoint,
+        change_feed=kafka_change_feed,
+        processor=form_processor,
+        change_processed_event_handler=MultiTopicCheckpointEventHandler(
+            checkpoint=checkpoint, checkpoint_frequency=100, change_feed=kafka_change_feed
```

Comment:
`checkpoint_frequency=100` means that locally we'll have to submit 100 forms before we see anything processed right? not for this PR, but I think in the future we should have this be dependent on whether on not this is in DEBUG mode. In DEBUG I think it should be 1